launch/docker_test.go

```diff
@@ -496,16 +496,12 @@ func TestHelperProcess(t *testing.T) {
 		}
 		os.Exit(0)
 	case "SUCCESS_TO_KILL":
-		if subcmd == "command" {
+		if subcmd == "command" || subcmd == "sleep" {
 			time.Sleep(fakeProcessLifeTime)
 			os.Exit(0)
 		}
-		os.Exit(0)
-	case "SUCCESS_TO_KILL_DOCKER_CMD":
-		if subcmd == "sleep" {
-			time.Sleep(fakeProcessLifeTime)
-		}
-	case "FAILED_TO_KILL":
+		os.Exit(1)
```

Comment:
This `os.Exit` means `sudo kill` failure
```suggestion
		os.Exit(0)
```

Reply:
thx.
I fixed.